examples/timestamping/backend/tests/config.rs

```diff
@@ -101,10 +104,11 @@ fn test_propose_configuration() {
     };
 
     // Propose valid configuration.
-    let new_config = propose_configuration(&mut testkit, config.clone())
-        .expect("Configuration proposal failed.");
-
-    assert_eq!(new_config.time_service_name, config.time_service_name);
+    let res = propose_configuration(&mut testkit, config);
+    assert!(
+        res.is_ok(),
+        format!("Configuration proposal failed: {:?}", res)
```

Comment:
This `assert` seems the same as calling `res.expect("Configuration proposal failed")`.